GroundDBMemoryPool/src/include/rdma_server.hh

```diff
@@ -1,12 +1,36 @@
 #pragma once
+#include <unordered_map>
 
 namespace mempool{
 
+/* The current design deploy unordered_map which is not memory continuous and thus
+inaccessible by one-sided RDMA. Further discussion and potential re-design are necessary. */
+class page_address_table {
+public:
+    std::unordered_map<KeyType, uintptr_t, KeyTypeHashFunction, KeyTypeEqualFunction> pat;
+    page_address_table(){}
+};
```

Comment:
Should PAT Table reuse LRU cache? LRU cache is am unordered map too.

Reply:
I am now a little confused about the `LRUCache<T, V>`, what `T` and `V` are we going to use?

Reply:
`T` and `V` `are `KeyType`  and `uintptr_t`. But `LRUCache` should refactor to use `KeyTypeHashFunction` and `KeyTypeEqualFunction`

Reply:
We could change `m` of `LRUCache` to a pointer to the `page_address_table` so that we can keep `LRUCache` untouched if `page_address_table` needs refactoring.

Reply:
Yes. `m` can be passed as an argument to LRUCache's constructor